rules/windows/persistence_suspicious_com_hijack_registry.toml

```diff
@@ -13,7 +13,7 @@ Identifies Component Object Model (COM) hijacking via registry modification. Adv
 executing malicious content triggered by hijacked references to COM objects.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "endgame-*"]
+index = ["logs-endpoint.events.*"]
```

Comment:
I removed Endgame support to check signer info; Endgame doesn't have this data outside process events.

Reply:
Good call